src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -317,7 +317,7 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                 files.AddRange(srcElement.Value.Trim(';').Split(';').Select(s => 
                     new ManifestFile
                     {
-                        Source = s.SafeTrim().TrimStart(slashes),
```

Comment:
I had a bug around leading slashes in the target.  I fixed that and made the assumption that a leading slash in the src would be the root of the project.  Clearly for Unix, it instead means a full path to a file.  For Windows, it also points to the root like if you started a path with "\folder".  I don't think anyone was using a leading slash to mean the project root so this won't break people.